### ISSUE TITLE
Final Project Peer Review

### ISSUE THREAD

**Issue by iarevalo** [s0ap/orie5741-project#14]
This project is about trying to predict the daily realized volatility of the S&P 500 Index. They not only used this data of the S&P 500 index, more specifically the 1-minute tick data for S&P 500, but also used indices for other markets, a recession indicator, and a few other external inputs to help predict this volatility. 

Pros:
- I like how they introduced more data and used the 1-minute tick data for S&P 500 making this fairly big data that they are truly working with. 
- I like how they decided to use a well-known model in the financial engineering world, the GARCH model, to compare the output of their Linear Regression outputs.
- I like how well and in-depth they described their exploratory data analysis section.
- I liked their visuals throughout the project as I found they provided good insight as to what they were doing and what kinds of results they provided.

Cons:
- I found the project to get kind of complicated to follow at times and that's given I have a bit of understanding of financial engineering so I believe that a non-financial engineering background person may have more trouble following at times and hence could be better described for such crowds.
- I do not understand why the team could not use a validation set since could you not just do a validation set such that its time period lies between the end of the training period and before the test period.
- They briefly discuss the implications of this work for the business context in the video but I did not see much written with regards to the business context in the paper.
- The report itself is too long going over by nearly a full page, and the video goes over by 8 minutes.

Overall, it was still a very interesting project and they seemed to have tackled it well.
